cli/support_test.go

```diff
@@ -156,11 +155,26 @@ func assertBundleContents(t *testing.T, path string) {
 		case "workspace/agent_startup_logs.txt":
 			bs := readBytesFromZip(t, f)
 			require.Contains(t, string(bs), "started up")
+		case "workspace/template.json":
+			var v codersdk.Template
+			decodeJSONFromZip(t, f, &v)
+			require.NotEmpty(t, v, "workspace template should not be empty")
+		case "workspace/template_version.json":
+			var v codersdk.TemplateVersion
+			decodeJSONFromZip(t, f, &v)
+			require.NotEmpty(t, v, "workspace template version should not be empty")
+		case "workspace/parameters.json":
+			var v []codersdk.TemplateVersionParameter
```

Comment:
`TemplateVersionParameter` or `WorkspaceBuildParameter`?

BTW There are also Template Version variables, but these may carry sensitive data.

Reply:
Ah, good catch. These are workspace build parameters. Will update 👍 

Agreed, we should leave out template version variables.